debugd/logstash/assets.go

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package logstash
+
+import "embed"
+
+// Assets are the exported Logstash template files.
+//
+//go:embed config/*
+//go:embed templates/*
+var Assets embed.FS
```

Comment:
Where is this used?

Reply:
https://github.com/edgelesssys/constellation/blob/660c9041174767232d762b0579ac06bafad0ff54/hack/logcollector/internal/filebeat.go#L23

Reply:
https://github.com/edgelesssys/constellation/blob/660c9041174767232d762b0579ac06bafad0ff54/hack/logcollector/internal/logstash.go#L23